### PR TITLE
Terse notes board meeting 2021-12

### PR DESCRIPTION
**Pull request by lenucksi**
Directors: Please all leave a PR review indicating acceptance or desire for changes.

### CODE REVIEW

**Review by rrrutledge — APPROVED** [review]
Thank you!

File: content/about/board/2021-12-02.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+show_meta: false
+title: '2021-12-02'
+---
+
+### Date and Time of Meeting
+
+2021-12-02, 7 p.m - 8 p.m. CEST / 12 p.m. - 1 p.m. CDT
```
Comment (gruetter) [inline content/about/board/2021-12-02.md]:
```suggestion
2021-12-02, 7 p.m. - 8 p.m. CET / 12 p.m. - 1 p.m. CDT
```

File: content/about/board/2021-12-02.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Call to order 19:05
+Adjourned 19:59
+Next meeting 13.1.2021 18:00 UTC
+
+### Role Call
```
Comment (gruetter) [inline content/about/board/2021-12-02.md]:
```suggestion
### Roll Call
```

File: content/about/board/2021-12-02.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+#### Directors Present  
+
+- Johannes Tigges (Assistant Secretary)
+- Isabel Drost-Fromm (President)
+- Daniel Izquierdo
```
Comment (gruetter) [inline content/about/board/2021-12-02.md]:
```suggestion
- Daniel Izquierdo Cortazar
```

File: content/about/board/2021-12-02.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- Max Capraro
+- Danese Cooper (Chair)
+
+#### Directors Absent
+- Jacob Green
+- Cedric Williams (Treasurer, proxied by Max)
+
+#### Members Present
+
+- Clare Dillon
```
Comment (gruetter) [inline content/about/board/2021-12-02.md]:
```suggestion
- Clare Dillon (Executive Director)
```

File: content/about/board/2021-12-02.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+show_meta: false
+title: '2021-12-02'
+---
+
+### Date and Time of Meeting
+
+2021-12-02, 7 p.m - 8 p.m. CEST / 12 p.m. - 1 p.m. CDT
+Call to order 19:05
```
Comment (gruetter) [inline content/about/board/2021-12-02.md]:
```suggestion
Call to order 7:05 p.m.
```

File: content/about/board/2021-12-02.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Adjourned 19:59
```
Comment (gruetter) [inline content/about/board/2021-12-02.md]:
```suggestion
Adjourned 7:59 p.m.
```

**Review by gruetter — APPROVED** [review]
Sorry for the nitpickyness, @lenucksi ;)

**Review by dicortazar — APPROVED** [review]
Thanks for the notes!

**Review by maxcapraro — APPROVED** [review]

**Review by rrrutledge — APPROVED** [review]
Looks great!  Thank you!

**Comment by rrrutledge** [comment]
Good to merge now?

**Comment by Danese** [comment]
Sorry to be late reviewing, but looks good to me.

**Review by Danese — APPROVED** [review]

**Review by MaineC — APPROVED** [review]

**Pull request merged**